### DISCUSSION TITLE
Option to create new pages in home directory automatically

### DISCUSSION THREAD

**Discussion by TheNomad11** [zim-desktop-wiki/zim-desktop-wiki#1675] [Ideas]
Let me explain: I am using the journal plugin and write down things I should do, f.ex make a plan for moving out of the flat. I then create a link and make a new page "plan for moving out" by highlighting the phrase and clicking CTL-L.  I create the new page, but this page is placed under journal/2021/10/10 by default. Wouldn*t it be better to have this new page placed under the home or main directory of this Zim-Notebook? Now this page is somehow hidden in the directoy structure? 

My suggestion would be an option to create new pages in the home or main directory of the notebook by default. What do you think?

**Comment by timelyfiasco** [comment]
When you click ctrl L you can add `:Home:plan for moving out`
But I personally prefer putting it under journal. You can use alt d to go to today's page. alt page up or down to navigate  notes from yesterday. 
![image](https://user-images.githubusercontent.com/33004592/136691833-1075b19a-ca8b-4395-ac52-7923c379c1d1.png)

> My suggestion would be an option to create new pages in the home or main directory of the notebook by default. What do you think?

Path to new pages are placed in the same directory to your opened page.
